coordinator/src/main/java/org/apache/uniffle/coordinator/CoordinatorConf.java

```diff
@@ -125,7 +125,21 @@ public class CoordinatorConf extends RssBaseConf {
       .stringType()
       .noDefaultValue()
       .withDescription("Remote Storage Cluster related conf with format $clusterId,$key=$value, separated by ';'");
-
+  public static final ConfigOption<String> COORDINATOR_REMOTE_STORAGE_SELECT_STRATEGY = ConfigOptions
+      .key("rss.coordinator.remote.storage.select.strategy")
+      .stringType()
+      .defaultValue("IO")
```

Comment:
Could we use "APP" as default value?

Reply:
Of course, you can let users choose by themselves, but I actually prefer the IO strategy, because if you use IO, you can avoid some risks better than the APP strategy.

Reply:
> Of course, you can let users choose by themselves, but I actually prefer the IO strategy, because if you use IO, you can avoid some risks better than the APP strategy.

For our production environment, one HDFS can't process all the the requests of rss shuffle servers, so we need to allocate the application to multiple HDFS by app nums.

Reply:
OK.